test/unit/src/audio/FftUnit.cpp

```diff
@@ -3,7 +3,7 @@
 // FIXME: OOURA roundtrip FFT seems to be broken on windows for sizeFft = 4 (https://github.com/cinder/Cinder/issues/1263)
 #if ! defined( CINDER_MSW )
 
-#include "catch.hpp"
+#include "../catch.hpp"

```

Comment:
Actually, I can think of a better place to put the catch header - at `cinder/test/unit/include/catch.hpp`. The benefit here that I see is that if other projects that already depend on cinder want to use catch.hpp, they can add that folder to their include path without picking up any of cinder's unit tests.  But the move can be done outside of this PR, and probably should as the xcode and VS project files will need to be updated.